<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>538</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Image Browser</string>
  </property>
  <widget class="QPushButton" name="pushButtonDownload">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSaveMeta">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Metadata</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAddLayer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add as a layer</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>371</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="lbTitle">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>TITLE:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayoutMetadata">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lb0">
        <property name="text">
         <string>PLATFORM:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbText0">
        <property name="text">
         <string>lbText0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb1">
        <property name="text">
         <string>ACQUISITION START:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbText1">
        <property name="text">
         <string>lbText1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb2">
        <property name="text">
         <string>ACQUISITION END:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbText2">
        <property name="text">
         <string>lbText2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lb3">
        <property name="text">
         <string>GSD:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lbText3">
        <property name="text">
         <string>lbText3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb4">
        <property name="text">
         <string>PROVIDER:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lbText4">
        <property name="text">
         <string>lbText4</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lb5">
        <property name="text">
         <string>FILE SIZE:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="lbText5">
        <property name="text">
         <string>lbText5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbThumbnail">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>Thumbnail</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
